test/e2e/exists_test.go

```diff
@@ -39,6 +39,7 @@ var _ = Describe("Podman image|container exists", func() {
 		Expect(session).Should(Exit(0))
 	})
 	It("podman image exists in local storage by short name", func() {
+		Skip("FIXME-8165: shortnames don't seem to work with quay (#8176)")
```

Comment:
@vrothberg FYI for short names work

Reply:
Thanks, Tom!

Reply:
@TomSweeneyRedHat @vrothberg late followup: this does not seem to be related to shortnames. #8176 is too weird for me to describe in a helpful title.